src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -333,7 +333,7 @@ public SourceText GetText(Encoding? encoding = null, SourceHashAlgorithm checksu
         /// <summary>
         /// Determine whether this node is structurally equivalent to another.
         /// </summary>
-        public bool IsEquivalentTo(SyntaxNode other)
+        public bool IsEquivalentTo([NotNullWhen(true)] SyntaxNode other)
```

Comment:
nit: consider adding `#nullable enable` block and marking `other` as annotated with `?`

Reply:
done!